conda_pypi/cli/pypi.py

```diff
@@ -0,0 +1,196 @@
+"""
+conda pypi subcommand for CLI
+"""
+
+from __future__ import annotations
+
+import argparse
+from pathlib import Path
+from logging import getLogger
+
+from conda.auxlib.ish import dals
+from conda.cli.conda_argparse import (
+    add_output_and_prompt_options,
+    add_parser_help,
+    add_parser_prefix,
+    _GreedySubParsersAction,
+)
+from conda.exceptions import ArgumentError
+from conda.base.context import context
+from conda.models.match_spec import MatchSpec
+
+from conda_pypi import convert_tree
+
+
+logger = getLogger(__name__)
+
+
+def configure_parser(parser: argparse.ArgumentParser):
+    add_parser_help(parser)
```

Comment:
We do not need to do this, conda adds the help arg for us:

https://github.com/conda/conda/blob/01167012a6ae1e1dfb73c6ac0902b7c8a9267a6f/conda/cli/conda_argparse.py#L324-L332